manifests/charts/global.yaml

```diff
@@ -568,3 +562,13 @@ version: ""
 
 meshConfig:
   enablePrometheusMerge: false
+  defaultConfig:
+    tracing:
+      tlsSettings:
+        mode: DISABLE # DISABLE, SIMPLE, MUTUAL, ISTIO_MUTUAL
```

Comment:
@rshriram @hanahmily can we support all 4 modes for this? I think only DISABLE/SIMPLE might actually work.

Reply:
For people running istio proxy next to zipkin, istio mutual helps.